__tests__/helpers/database-helpers.ts

```diff
@@ -15,6 +15,7 @@ export class DatabaseHelpers {
     const uri = this.server.getUri();
 
     await mongoose.connect(uri);
+    await mongoose.connection.syncIndexes();
```

Comment:
@TaiKamilla 

some tests failed due to this missing line, when being pushed to github